quickstarts/spring-with-s2i.yaml

```diff
@@ -0,0 +1,208 @@
+  introduction: >-
+    **Spring** is a Java framework for building applications based on a distributed microservices architecture. 
+    
+    - Spring enables easy packaging and configuration of Spring applications into a self-contained executable application which can be easily deployed as a container to OpenShift.
+    
+    - Spring applications can integrate OpenShift capabilities to provide a natural "Spring on OpenShift" developer experience for both existing and net-new Spring applications. For example:
+    
+    - Externalized Configuration using Kubernetes ConfigMaps and integration with Spring Cloud Kubernetes
+    
+    - Service Discovery using Kubernetes Services
+    
+    - Load Balancing with Replication Controllers
+    
+    - Kubernetes Health Probes and integration with Spring Actuator
+    
+    - Metrics: Prometheus, Grafana, and integration with Spring Cloud Sleuth
+    
+    - Distributed Tracing with Istio & Jaeger Tracing
+    
+    - Developer tooling through Red Hat OpenShift and Red Hat CodeReady developer tooling to quickly scaffold new Spring projects, gain access to familiar Spring APIs in your favorite IDE, and deploy to Red Hat OpenShift
+
+  tasks:
+    - description: >-
+        To create a Spring application:
+        
+        1. Click on the [perspective switcher]{{highlight qs-perspective-switcher}} at the top of the navigation, and select **Developer**.
```

Comment:
```suggestion
        1. Enter the Developer perspective: In the main navigation, click the dropdown menu and select **Developer**.
```

Reply:
sticking with perspective switcher to be consistent with dev console docs